jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -292,4 +292,17 @@ private void startWebSocketConnection() {
             logger.error("No websocket url returned by Slack.");
         }
     }
+
+    /**
+     * Method to reconnect to websocket
+     *
+     */
+    public void reconnect() {
+    	if (slackService.getWebSocketUrl() != null) {
```

Comment:
Code is repeated. Maybe we can pull out the common code from `startWebSocketConnection` and `reconnect` methods to another method.

Reply:
That's a good point. Renamed it to a public openConnection method and removed duplicate code. Now it is logically named for both locations.